<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventFeedPage</class>
 <widget class="QWidget" name="EventFeedPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="6">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabClientFeed">
      <attribute name="title">
       <string>Питание пациента</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CFeedInDocTableView" name="tblClientFeed">
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPatronFeed">
      <attribute name="title">
       <string>Питание лица по уходу</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="1" column="0" colspan="2">
        <widget class="CFeedInDocTableView" name="tblPatronFeed"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="lblPatronName">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblCreatePerson">
     <property name="text">
      <string>Автор: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblCreateDate">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lblModifyPerson">
     <property name="text">
      <string>Изменил: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="lblModifyDate">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="btnFeedPrint">
     <property name="text">
      <string>Печать</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="btnGetMenu">
     <property name="text">
      <string>Шаблон</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CFeedInDocTableView</class>
   <extends>QTableView</extends>
   <header>Events/EventFeedModel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>tblClientFeed</tabstop>
  <tabstop>tblPatronFeed</tabstop>
  <tabstop>btnFeedPrint</tabstop>
  <tabstop>btnGetMenu</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
